src/client/App.js

```diff
@@ -21,6 +27,23 @@ function App() {
         <Route exact path="/">
           <Home />
         </Route>
+
+        <Route exact path="/login-page">
```

Comment:
We don't usually put `-page` in URLs, `/login` is fine.

Reply:
> We don't usually put `-page` in URLs, `/login` is fine.

updated URL names